typescript/phoenix-sdk/src/utils/genericTokenMint.ts

```diff
@@ -61,3 +62,42 @@ export function getAirdropSplInstruction(
 
   return createAirdropSplInstruction(accounts, amount);
 }
+
+// For devnet testing, we use a dummy devnet market whose base and quote tokens are mintable through these instructions.
```

Comment:
add: "This function takes in a market and a trader pubkey and airdrops both tokens for that market to the trader."